OpenRA.Mods.Common/Traits/StoresResources.cs

```diff
@@ -47,8 +47,10 @@ void INotifyOwnerChanged.OnOwnerChanged(Actor self, Player oldOwner, Player newO
 		void INotifyCapture.OnCapture(Actor self, Actor captor, Player oldOwner, Player newOwner, BitSet<CaptureType> captureTypes)
 		{
 			var resources = Stored;
-			oldOwner.PlayerActor.Trait<PlayerResources>().TakeResources(resources);
-			newOwner.PlayerActor.Trait<PlayerResources>().GiveResources(resources);
+			var oldpr = oldOwner.PlayerActor.Trait<PlayerResources>();
+			var newpr = newOwner.PlayerActor.Trait<PlayerResources>();
+			oldpr.TakeResources(resources);
+			newpr.GiveResources(resources);
```

Comment:
Revert this. You are using abbreviated variable names here and elsewhere which make this needlessly cryptic.

Reply:
fixed.